### PR TITLE
Add step to source setup file in README

### PR DESCRIPTION
**Pull request by ihsan314**
### Description
*Give a high-level description of the changes in this PR*

### Testing Done
*Outline any testing that was done for these changes. This could be unit tests, integration tests, etc.*

### Resolved Issues
*Link any issues that this PR resolved. Eg `resolves #1, #2, and #5` (note that they MUST be specified like this so Github can automatically close them then this PR merges)*

### Review Checklist
*(Please check every item to indicate your code complies with it (by changing `[ ]`->`[x]`). This will hopefully save both you and the reviewer(s) a lot of time!)*
***It is the reviewers responsibility to also make sure every item here has been covered***
- [ ] **Start of document comments**: each `.cpp` and `.h` file should have a comment at the start of it. See files in `src/sample_package` for examples.
- [ ] **Function comments**: All function definitions (usually in the `.h` file) should have a javadoc style comment at the start of them. For examples, see the classes defined in `src/sample_package`
- [ ] **Remove all commented out code**
- [ ] **Remove extra print statements**: for example, those just used for testing
- [ ] **Resolve all TODO's**: All `TODO` (or similar) statements should either be completed or associated with a github issue
***Feel free to make additions of things that we should be checking to this file if you think there's something missing!!!!***

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,8 +45,9 @@ if you're on campus use the `ubcsecure` or `resnet` networks for best results.
 7. Clone your server-side repository from the terminal by running `git clone --recursive https://github.com/YOUR_USERNAME/Snowflake.git` (`YOUR_USERNAME` is your github username)
 8. To start the setup run `cd ~/Snowflake && ./setup_scripts/install_tools.sh` (Do not run this script as root).
     - *Just choose yes and enter your password when the terminal prompts you*
-9. To build the workspace run `catkin_make`. If everything compiles correctly and you don't get any errors, then you're good to go!
-10. If catkin_make fails, and mentions missing .cmake files, run `./setup_scripts/install_dependencies.sh`.
+9. Run `source /opt/ros/melodic/setup.sh`.
```
Comment (ihsan314) [inline README.md:48]: Comment for illustrative purposes

File: README.md [hunk 2]
```diff
@@ -45,8 +45,9 @@ if you're on campus use the `ubcsecure` or `resnet` networks for best results.
...
+10. To build the workspace run `catkin_make`. If everything compiles correctly and you don't get any errors, then you're good to go!
```
Comment (ihsan314) [inline README.md:49]: Different comment, also for illustrative purposes

**Review by ihsan314 — COMMENTED** [review]
Another illustrative comment, this time in the review section

File: README.md [hunk 3]
```diff
@@ -45,8 +45,9 @@ if you're on campus use the `ubcsecure` or `resnet` networks for best results.
...
+11. If catkin_make fails, and mentions missing .cmake files, run `./setup_scripts/install_dependencies.sh`.
```
Comment (ihsan314) [inline README.md:50]: Comment on multiple lines

**Review by Roozki — APPROVED** [review]
Looks good to me, no typos and ros command is proper.

**Pull request merged**